roles/custom/matrix-media-repo/templates/media-repo/systemd/matrix-media-repo.service.j2

```diff
@@ -0,0 +1,52 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Matrix media-repo
+{% for service in matrix_media_repo_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_media_repo_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+DefaultDependencies=no
+
+[Service]
+Type=simple
+Environment="HOME={{ devture_systemd_docker_base_systemd_unit_home_path }}"
+ExecStartPre=-{{ devture_systemd_docker_base_host_command_sh }} -c '{{ devture_systemd_docker_base_host_command_docker }} kill matrix-media-repo 2>/dev/null || true'
+ExecStartPre=-{{ devture_systemd_docker_base_host_command_sh }} -c '{{ devture_systemd_docker_base_host_command_docker }} rm matrix-media-repo 2>/dev/null || true'
+
+ExecStartPre={{ devture_systemd_docker_base_host_command_docker }} create \
+			--rm \
+			--name=matrix-media-repo \
+			--log-driver=none \
+			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
+			--cap-drop=ALL \
+			--network={{ matrix_docker_network }} \
+			-p {{ matrix_media_repo_port }}:{{ matrix_media_repo_port }} \
+			{% if matrix_media_repo_metrics_enabled %}
+			-p {{ matrix_media_repo_metrics_port }}:{{ matrix_media_repo_metrics_port }} \
```

Comment:
Here, we may also wish to not always expose the metrics publicly on a given port (on all interfaces).

If the integrated (into the playbook) Prometheus server is used, then it can scrape metrics over the container network. `matrix_prometheus_services_connect_scraper_media_repo_static_configs_target` is already configured correctly for this.

Introducing another `_bind_port` variable (with a default `""` value) here and only exposing the port if it's truthy will be better.